tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,26 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    tai_new_inventory = [item_a, item_b, item_f]
+    jesse_new_inventory = [item_c, item_d, item_e]
+
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
+    assert result is True
     # - That tai and jesse's inventories are the correct length
+    assert len(tai.inventory) == 3 and len(jesse.inventory) == 3
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    # tai should have a, b, f and jesse should have c, d, e
+    for item in jesse_new_inventory:
+        assert item in jesse.inventory
+    for item in tai_new_inventory:
+        assert item in tai.inventory
```

Comment:
These are all great! You could also add assertions that `item_c` isn't in tai's inventory and that `item_f` isn't in jesse's